mod/search/start.php

```diff
@@ -316,7 +316,7 @@ function search_remove_ignored_words($query, $format = 'array') {
 	//$query = str_replace(array('"', '-', '+', '~'), '', stripslashes(strip_tags($query)));
 	$query = stripslashes(strip_tags($query));
 	
-	$words = explode(' ', $query);
+	$words = preg_split('/\s+/', $query);
```

Comment:
You should also trim() the $query above. Otherwise you need to add PREG_SPLIT_NO_EMPTY.